### PR TITLE
Thanks page

### PR DESCRIPTION
**Pull request by TarasKun**
Create Thanks page component. Add styles, route "/thanks". Change theme, and language is working.

### CODE REVIEW

**Comment by TarasKun** [comment]
> You pulling into master, but should to development
Change master to development

File: src/pages/thanks-page/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import Thanks from './thanks-page';
```
Comment (Alan18081) [inline src/pages/thanks-page/index.js:1]: Please, rename `Thanks` to `ThanksPage` because the name of the exported component and the name of the file should match

File: src/store/store.js [hunk 2]
```diff
@@ -11,11 +11,7 @@ const configureStore = () => {
   const sagaMiddleware = createSagaMiddleware();
   const store = createStore(
     rootReducer(history),
-    compose(
-      applyMiddleware(routerMiddleware(history), sagaMiddleware),
-      window.__REDUX_DEVTOOLS_EXTENSION__ &&
```
Comment (Alan18081) [inline src/store/store.js:16]: Please, revert this change because we need redux devtools in development

**Comment by Alan18081** [comment]
Also, please, rename image `Rectangle` to `rectangle`

File: src/pages/thanks-page/index.js (hunk 1 above)
Comment (TarasKun) [reply src/pages/thanks-page/index.js:1]: Done

File: src/store/store.js (hunk 2 above)
Comment (TarasKun) [reply src/store/store.js:16]: Done
